tests/README.md

```diff
@@ -156,6 +156,7 @@ not known until test time. The following runtime variables are available for the
 
 * `{{BPFTRACE}}`: Path to bpftrace executable
 * `{{BEFORE_PID}}`: Process ID of the process in the first `BEFORE` directive
+* `{{EXPECT_NCPUS}}`: The number of CPUs in all `EXPECT*` directive
```

Comment:
we shouldn't make this a new directive, let's just use the output from a command line tool in the runtime test. If that's too much of a pain then we can forego the runtime test as this is a pretty straightforward addition.